src/cascade/dismod/db/metadata.py

```diff
@@ -0,0 +1,519 @@
+"""
+This describes the tables in the sqlite file that Dismod reads.
+"""
+import enum
+import logging
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Float, Enum, ForeignKey
+
+
+LOGGER = logging.getLogger("cascade_at.dm_file_format.dmtables")
```

Comment:
Maybe we want to change the "cascade_at..." part?

Reply:
I have fallen into my own trap. This is why I like `logging.getLogger(__name__)`